src/Sulu/Component/Rest/ListBuilder/AbstractListBuilder.php

```diff
@@ -238,8 +238,19 @@ public function search($search)
      */
     public function sort(AbstractFieldDescriptor $fieldDescriptor, $order = self::SORTORDER_ASC)
     {
-        $this->sortField = $fieldDescriptor;
-        $this->sortOrder = $order;
+        $this->sortFields = array($fieldDescriptor);
```

Comment:
Why does this have to be changed? And in case there is a reason, why didn't you write here `$this->sortFields[] = $fieldDescriptor;` and therefore omit `addOrderBy`?